sound/soc/sof/intel/cnl.c

```diff
@@ -233,12 +234,26 @@ void cnl_ipc_dump(struct snd_sof_dev *sdev)
 
 static int cnl_register_clients(struct snd_sof_dev *sdev)
 {
-	return intel_register_ipc_test_clients(sdev);
+	int ret;
+
+	ret = intel_register_ipc_test_clients(sdev);
+	if (ret < 0)
+		return ret;
+
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA_PROBES)
+	return sof_client_dev_register(sdev, "probes", 0);
```

Comment:
not sure if that would be an improvement, but seeing how we first register ipc-flood with id=1, now probes with id=0... Maybe define macros like
```
#define IPC_FLOOD_ID_0 0x100
#define PROBES_ID_0 0x200
```
etc.?

Reply:
...to clarify this a bit, I meant that in that case further IPC flood IDs would be 0x101, 0x102, etc. (I think we currently use 2 of them per platform).